_i18n/ar/resources/user-guides/make-payment.md

```diff
@@ -0,0 +1,80 @@
+
+4) Click on `Add recipient` if you want to add more recipients to the transaction. A transaction with multiple recipients will only pay a single transaction fee, but the higher the number of recipients, the higher the fee will be.
+
+Sending to multiple recipients doesn't compromise your privacy, since each recipient will not know the other recipients of the transaction, nor how much was sent for each one.
+
+5) If you are using Monero GUI in Advanced mode, you will see a `Transaction priority` dropbox. It's recommended to leave it on `Automatic`.
+
+For privacy reasons, it's not possible to enter a custom fee, but you can choose between four fee levels. It's not necessary to change the fee level unless the Monero network is congested.
+
+6) Optionally, click on `Add description` to add a note for your transaction, which will be saved in your wallet and can be later consulted on `History` page. This feature is for your control only: it will not be sent to the receiver neither registered on the blockchain.
+
+7) Click on `Send` button.
+
+8) A transaction confirmation dialog will appear. Check if the transaction details are correct and click on `Confirm` button.
+
+<img src="/img/resources/user-guides/en/make-payment/tx_confirmation.png" style="width: 600px;"/>
+
+9) Enter your wallet password and click on `Send transaction` button.
+
+<img src="/img/resources/user-guides/en/make-payment/password.png" style="width: 427px;"/>
+
+10) Next, you will see the `Transaction successfully sent!` dialog.
+
+<img src="/img/resources/user-guides/en/make-payment/tx_successful.png" style="width: 600px;"/>
+
+This means your transaction was sent to the Monero network and is now waiting to be confirmed (included in a @block) by a miner. Now, you just have to wait for your transaction to be confirmed, which takes on average 2 minutes. Once you've sent your transaction to the network, there is nothing you can do to make the confirmation process faster.
+
+This dialog will also display the `Transaction ID` (txid), which is a unique identifier for your transaction. Optionally, click on `Copy` button to copy it to the clipboard.
+
+Click on `View progress` button to open `Transactions` page and see the transaction details. On this page you can monitor the @blockchain and check if your transaction was confirmed.
+
+### How to check if your payment transaction was confirmed
+After clicking on `View progress` button, `Transactions` page will open with your latest transaction selected.
+
+<img src="/img/resources/user-guides/en/make-payment/transactions_page.png" style="width: 600px;"/>
+
+There are three possible states for your sent transaction:
+- **Sent (Pending)**: your transaction was sent to the Monero network but it is still waiting to be confirmed. Please wait for a while until a miner includes your transaction in a block, confirming it. This process takes on average 2 minutes, but sometimes can take longer.
```

Comment:
Add a newline between these two lines.

Reply:
Why?

Reply:
po4a won't parse it correctly because they are two different markdown entities.